src/java.base/share/classes/java/io/File.java

```diff
@@ -1376,7 +1376,10 @@ public boolean mkdirs() {
      * file from one filesystem to another, it might not be atomic, and it
      * might not succeed if a file with the destination abstract pathname
      * already exists.  The return value should always be checked to make sure
-     * that the rename operation was successful.
+     * that the rename operation was successful.  As instances of {@code File}
+     * are immutable, the abstract pathname represented by this {@code File}
+     * object does not itself change although the filesystem object it denoted
+     * might have moved.
      *
```

Comment:
Is there a way to mix in a mention of the {@ code "dest"} argument.
"might have moved to the abstract pathname provided in the dest argument".

Reply:
Good idea.